app/build.gradle

```diff
@@ -43,15 +46,15 @@ android {
 }
 
 dependencies {
-    implementation'androidx.activity:activity:1.6.1'
+    implementation 'androidx.activity:activity:1.8.0'
     testImplementation 'junit:junit:4.13.2'
-    androidTestImplementation 'androidx.test.espresso:espresso-core:3.4.0'
-    androidTestImplementation 'androidx.test:runner:1.4.0'
-    androidTestImplementation 'androidx.test:rules:1.4.0'
-    androidTestImplementation 'androidx.test.espresso:espresso-intents:3.4.0'
-    androidTestImplementation 'androidx.test.espresso:espresso-contrib:3.4.0'
-    androidTestImplementation 'androidx.test.ext:junit:1.1.3'
-    androidTestUtil 'androidx.test:orchestrator:1.4.1'
+    androidTestImplementation 'androidx.test.espresso:espresso-core:3.5.1'
+    androidTestImplementation 'androidx.test:runner:1.5.2'
+    androidTestImplementation 'androidx.test:rules:1.5.0'
+    androidTestImplementation 'androidx.test.espresso:espresso-intents:3.5.1'
+    androidTestImplementation 'androidx.test.espresso:espresso-contrib:3.5.1'
+    androidTestImplementation 'androidx.test.ext:junit:1.1.5'
+    androidTestUtil 'androidx.test:orchestrator:1.4.2'
 
     ext.okHttpVersion = '4.9.0'
     implementation 'com.squareup.okhttp3:okhttp:' + ext.okHttpVersion
```

Comment:
Since we are updating dependencies, can we upgrade it to the latest stable version of okhttp [here](https://mvnrepository.com/artifact/com.squareup.okhttp3/okhttp/4.11.0)